projects/rocprofiler-sdk/source/lib/rocprofiler-sdk/registration.cpp

```diff
@@ -750,6 +737,13 @@ invoke_client_finalizer(rocprofiler_client_id_t client_id)
 }
 }  // namespace
 
+struct attach_status*
+get_attach_status()
+{
+    static auto*& _v = common::static_object<attach_status>::construct(false);
```

Comment:
```suggestion
    static auto*& _v = common::static_object<attach_status>::construct();
```
Use the default member values